### PR TITLE
Add apps to the server's startup verification list

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build #sp#awn.fullfat.bucket=com.ibm.ws.cdi.2.0_fat, com.ibm.ws.cdi.annotations_fat, com.ibm.ws.cdi.api_fat, com.ibm.ws.cdi.beansxml_fat, com.ibm.ws.cdi.beansxml.implicit_fat, com.ibm.ws.cdi.ejb_fat, com.ibm.ws.cdi.extension_fat, com.ibm.ws.cdi.jee_fat, com.ibm.ws.cdi.jndi_fat, com.ibm.ws.cdi.lifecycle_fat, com.ibm.ws.cdi.noncontextual_fat, com.ibm.ws.cdi.third.party_fat, com.ibm.ws.cdi.visibility_fat

**Comment by benjamin-confino** [comment]
New build at https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_h3sP8AeHEeqfUosHDHrRQQ

**Comment by benjamin-confino** [comment]
#run-libby-bot

File: dev/fattest.simplicity/src/com/ibm/ws/fat/util/ShrinkWrapSharedServer.java [hunk 1]
```diff
@@ -193,6 +194,8 @@ protected void before() {
             for (String path : archivesAndPaths.get(archive)) {
                 try {
                     ShrinkHelper.exportToServer(getLibertyServer(), path, archive);
+                    String appName = archive.getName().substring(0, archive.getName().length() - 4); //remove the file type suffix
+                    getLibertyServer().addInstalledAppForValidation(appName);
```
Comment (Azquelt) [inline dev/fattest.simplicity/src/com/ibm/ws/fat/util/ShrinkWrapSharedServer.java]: Can you be sure that an app deployed here is meant to start?

This seems like something only the test would know.

File: dev/fattest.simplicity/src/com/ibm/ws/fat/util/ShrinkWrapSharedServer.java [hunk 2]
```diff
@@ -39,10 +39,11 @@
      * Creates a {@link SharedServer} then runs any methods in testClass annotated with
      * {@link BuildShrinkWrap} and copies the returned Archives to the servers
      * <p>
-     * Methods must be static, have no parameters, return: {@code Archive}, {@codeArchive[]}, {@code List<Archive>} or {@code Map<Archive,String>}.
+     * Methods must be static, have no parameters, return: {@code Archive}, {@codeArchive[]}, {@code List<Archive>}, {@code Map<Archive,String>}.
+     * or {@code Map<Archive,List<String>>}.
      * If {@code Archive}, {@code}List<Archive> or Archive[] is returned the returned values will be placed to the server's
-     * dropins folders. If a map is returned each archive will be placed wherever the
-     * string points too.
```
Comment (Azquelt) [inline dev/fattest.simplicity/src/com/ibm/ws/fat/util/ShrinkWrapSharedServer.java]: This doc update doesn't seem related to the code change. I assume you're just correcting the existing doc?

Comment (benjamin-confino) [reply dev/fattest.simplicity/src/com/ibm/ws/fat/util/ShrinkWrapSharedServer.java]: Correct

**Comment by benjamin-confino** [comment]
#run-libby-bot
